### DISCUSSION TITLE
Relationship Search

### DISCUSSION THREAD

**Discussion by emcphersonburke** [wpengine/atlas-content-modeler#617] [Ideas]
Relationships are great, but linking isn't currently practical when you've got more than a handful of possible choices. For example, if I have a Book model and an Author model and they're linked via a Relationship, I have to click through all the entries, five at a time, until I find the Author I'm looking for. This becomes unwieldy as the number of Author objects increases.

Adding a search mechanism would greatly improve this experience.

**Comment by derekperkins** [comment]
This is very important for us
